jdk/src/share/bin/jli_util.h

```diff
@@ -101,4 +107,32 @@ void     JLI_TraceLauncher(const char* fmt, ...);
 void     JLI_SetTraceLauncher();
 jboolean JLI_IsTraceLauncher();
 
+/*
+ * JLI_List - a dynamic list of char*
+ */
+struct JLI_List_
+{
+    char **elements;
+    size_t size;
+    size_t capacity;
+};
+typedef struct JLI_List_ *JLI_List;
+
+JLI_List JNICALL
+JLI_List_new(size_t capacity);
+
+void JLI_List_free(JLI_List l);
+void JLI_List_ensureCapacity(JLI_List l, size_t capacity);
+
+/* e must be JLI_MemFree-able */
+void JNICALL
+JLI_List_add(JLI_List l, char *e);
+
+/* a copy is made out of beg */
+void JLI_List_addSubstring(JLI_List l, const char *beg, size_t len);
+char *JLI_List_combine(JLI_List sl);
+
+jboolean JNICALL
+JLI_ParseOpenJ9ArgsFromEnvVar(JLI_List args, const char *var_name);
```

Comment:
`JNICALL` is only necessary for exported functions; suggest you remove them from the declarations here, and from the function definitions.